crates/solver/src/settlement_submission/submitter.rs

```diff
@@ -474,16 +485,22 @@ impl<'a> Submitter<'a> {
             tx.clone().access_list(access_list.clone()).estimate_gas()
         )?;
 
+        ensure!(
+            gas_before_access_list > gas_after_access_list,
+            "access list exist but does not lower the gas usage"
+        );
+        let gas_percent_saved = (gas_before_access_list.to_f64_lossy()
+            - gas_after_access_list.to_f64_lossy())
+            / gas_before_access_list.to_f64_lossy()
+            * 100.;
         tracing::debug!(
-            "gas before/after access list: {}/{}, access_list: {:?}",
+            "gas before/after access list: {}/{}, access_list: {:?}, gas percent saved: {}",
             gas_before_access_list,
             gas_after_access_list,
             access_list,
+            gas_percent_saved
         );
-        ensure!(
-            gas_before_access_list > gas_after_access_list,
-            "access list exist but does not lower the gas usage"
-        );
+        track_access_list_gas_saved(gas_percent_saved);
```

Comment:
In general yes, but I was aiming to measure/compare only the success of access list estimators (now we are on tenderly but will switch to node).

In case we want to know the real onchain effect of access lists, I guess I would need to do that via tenderly simulation (because nodes do not support simulation with `transaction_index`).

Reply:
OK - makes sense that we will always approximate. But still shouldn't we only count this if we see a mined transaction? My reasoning is that, for example, we may count the gas saved from a Flashbots transaction timeout, even though its 0 - and that will get us misrepresented (higher) savings metric than we actually get.

I agree that the complexity of simulating at transaction index to get the exact amount is not necessarily worth it though.

Reply:
Added metric for the mined transactions only. One concern I have now is that our metric (now based on tenderly) will be skipped if tenderly sync is behind our node sync. Let's see.